webpack.config.js

```diff
@@ -0,0 +1,38 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+'use strict';
+
+//@ts-check
+
+// See https://github.com/Microsoft/vscode-azuretools/wiki/webpack for guidance
+
+const process = require('process');
+const webpack = require('webpack');
+const StringReplacePlugin = require("string-replace-webpack-plugin");
+const dev = require("vscode-azureextensiondev");
+
+let DEBUG_WEBPACK = !!process.env.DEBUG_WEBPACK;
+
+let config = dev.getDefaultWebpackConfig({
```

Comment:
The `vscode-azureappservice` npm package has icons just like the `vscode-azureextensionui` package does. Did a quick spot check and saw:
![screen shot 2019-01-23 at 8 44 58 am](https://user-images.githubusercontent.com/11282622/51622401-64df1480-1eeb-11e9-97d0-9de5ae0e4dbc.png)

Don't care if you fix in this PR or file a bug to fix later

Reply:
https://github.com/Microsoft/vscode-azureappservice/issues/833, thanks